index.js

```diff
@@ -373,62 +373,86 @@
   ];
 
   //  ValidDate :: Type
-  $.ValidDate = $.NullaryType(
+  $.ValidDate = NullaryType(
     'sanctuary-def/ValidDate',
     function(x) { return $.Date.test(x) && !isNaN(x.valueOf()); }
   );
 
+  //  PositiveNumber :: Type
+  $.PositiveNumber = NullaryType(
+    'sanctuary-def/PositiveNumber',
+    function(x) { return $.Number.test(x) && x > 0; }
+  );
+
+  //  NegativeNumber :: Type
+  $.NegativeNumber = NullaryType(
+    'sanctuary-def/NegativeNumber',
+    function(x) { return $.Number.test(x) && x < 0; }
+  );
+
   //  ValidNumber :: Type
-  $.ValidNumber = $.NullaryType(
+  var ValidNumber = $.ValidNumber = NullaryType(
     'sanctuary-def/ValidNumber',
     function(x) { return $.Number.test(x) && !isNaN(x); }
   );
 
+  //  NonZeroValidNumber :: Type
+  $.NonZeroValidNumber = NullaryType(
```

Comment:
Good question. I see the appeal, but I wonder whether it might only be used in situations where the author *intended* to use `NonZeroValidNumber`. In other words, would we be setting a trap by giving such a convenient name to such an "unlikely" type?